.circleci/config.yml

```diff
@@ -147,10 +147,15 @@ orbs:
             type: string
             description: Database to use
             default: mnesia
+          tls_dist:
+            type: string
```

Comment:
This looks like a boolean.

Reply:
yeah but there are multiple places in the project where "yes"/"no" values are considered:
https://github.com/esl/MongooseIM/blob/master/big_tests/Makefile#L20
https://github.com/esl/MongooseIM/blob/master/tools/test-runner.sh#L295
https://github.com/esl/MongooseIM/blob/master/tools/travis-common-vars.sh#L14

should I change it everywhere ?

Reply:
IMHO, `true/false` sounds better and fits `boolean` type if some tools support it (like this config). So yeah, I would change it everywhere.